src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
```

Comment:
```suggestion
public class Lottery {
```
тут нема сенсу використовувати наслідування